### PR TITLE
Py3 only

### PR DESCRIPTION
**Pull request by jenskutilek**

### CODE REVIEW

**Comment by typemytype** [comment]
py3.5 is not supported by fontTools and will raises:  `No matching distribution found for fonttools[lxml,type1,ufo,unicode,woff]>=4.0.0 (from ufo-extractor==0.3.1.dev0)`

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 sudo: false
 language: python
 python:
-  - 2.7
   - 3.5
```
Comment (typemytype) [inline .travis.yml:4]: test on 3.5 can be removed or replaced by 3.6

**Review by typemytype — CHANGES_REQUESTED** [review]

**Review by jenskutilek — COMMENTED** [review]
Thanks!

**Comment by typemytype** [comment]
thanks Jens!

**Pull request merged**